src/hotspot/share/cds/cds_globals.hpp

```diff
@@ -94,6 +94,10 @@
            "(2) always map at preferred address, and if unsuccessful, "     \
            "do not map the archive")                                        \
            range(0, 2)                                                      \
+                                                                            \
+  product(bool, ArchiveParallelRelocation, true, DIAGNOSTIC,                \
+          "Use parallel relocation code to speed up startup.")              \
```

Comment:
Maybe wait after JEP 483 integration and rename this flag to `AOTCacheParallelRelocation` to be consistent with the new naming scheme?

Reply:
Yeah, we can do that. Actually, I will rename the flag right now.